hieradata/node/comcam-mcm.cp.lsst.org.yaml

```diff
@@ -1,37 +1,89 @@
 ---
-network::interfaces_hash:
-  em1:  # fqdn
-    bootproto: "dhcp"
-    defroute: "yes"
-    onboot: "yes"
-    type: "Ethernet"
-  em2:  # dds
-    bootproto: "none"
-    defroute: "no"
-    ipaddress: "139.229.170.11"
-    netmask: "255.255.255.0"
-    nozeroconf: "yes"
-    onboot: "yes"
-    type: "Ethernet"
-  em3:  # not connected
-    bootproto: "none"
-    onboot: "no"
-    type: "Ethernet"
-  em4:  # not connected
-    bootproto: "none"
-    onboot: "no"
-    type: "Ethernet"
-  p1p1:  # not connected
-    bootproto: "none"
-    onboot: "no"
-    type: "Ethernet"
-  p1p2:  # not connected
-    bootproto: "none"
-    onboot: "no"
-    type: "Ethernet"
+nm::connections:
+  eno1:
+    content:
+      connection:
+        id: "eno1"
+        uuid: "03da7500-2101-c722-2438-d0d006c28c73"
+        type: "ethernet"
+        interface-name: "eno1"
+      ethernet: {}
+      ipv4:
+        method: "auto"
+      ipv6:
+        method: "disabled"
+      proxy: {}
+  eno2:  #dds interface
+    content:
+      connection:
+        id: "eno2"
+        uuid: "03da7500-2101-c722-2438-d0d006c28c73"
+        type: "ethernet"
+        interface-name: "eno2"
+      ethernet: {}
+      ipv4:
+        address1: "139.229.170.11/24,139.229.170.254"
```

Comment:
Please add this IP to ipam.

Reply:
IP was reserved and added to IPAM.

Reply:
https://ipam.cp.lsst.org/index.php?page=subnets&section=1&subnetId=23